enterprise/backup/src/main/java/org/neo4j/backup/BackupTool.java

```diff
@@ -63,23 +63,24 @@
     private static final String VERIFY = "verify";
     private static final String CONFIG = "config";
 
+    private static final String CONSISTENCY_CHECKER = "consistency-checker";
 
     private static final String TIMEOUT = "timeout";
     private static final String FORENSICS = "gather-forensics";
     public static final String DEFAULT_SCHEME = "single";
     static final String MISMATCHED_STORE_ID = "You tried to perform a backup from database %s, " +
-            "but the target directory contained a backup from database %s. ";
+                                              "but the target directory contained a backup from database %s. ";
 
     static final String WRONG_FROM_ADDRESS_SYNTAX = "Please properly specify a location to backup in the" +
-            " form " + dash( HOST ) + " <host> " + dash( PORT ) + " <port>";
+                                                    " form " + dash( HOST ) + " <host> " + dash( PORT ) + " <port>";
 
     static final String UNKNOWN_SCHEMA_MESSAGE_PATTERN = "%s was specified as a backup module but it was not found. " +
             "Please make sure that the implementing service is on the classpath.";
 
     static final String NO_SOURCE_SPECIFIED = "Please specify " + dash( HOST ) + " and optionally " + dash( PORT ) +
-            ", examples:\n" +
-            "  " + dash( HOST ) + " 192.168.1.34\n" +
-            "  " + dash( HOST ) + " 192.168.1.34 " + dash( PORT ) + " 1234";
+                                              ", examples:\n" +
```

Comment:
It's the jar from the intranet. Since everything else is out of date there, perhaps the jar is too.